docs/components/input-controller/_index.md

```diff
@@ -15,33 +15,26 @@ hide_children: true
 # SME: James
 ---
 
-You are likely already familiar with human-interface devices, like keyboards and mice, elevator button panels, light power switches, joysticks, and gamepads, or, video game controllers, from your daily life.
+Input controllers provide an API for configuring callbacks for events, allowing you to configure input devices to control your machines.
 
-Configuring an _input_ component allows you to use devices like these with your machine, enabling you to control your machine's actions by interacting with the device.
+If you have a keyboards, mice, elevator button panels, light power switches, joysticks, and gamepads, or, video game controllers with which you want to control a robotic base, use an _input controller_ component.
 
-This component currently supports devices like gamepads and joysticks that contain one or more [Controls](#control-field) representing the individual axes and buttons on the device.
-To use the controller's inputs, you must [register callback functions](/components/input-controller/#registercontrolcallback) to the [Controls](#control-field) with the `input` API.
+This component supports devices like gamepads and joysticks that contain one or more [`Control`s](/appendix/apis/components/input-controller/#control-field) representing the individual axes and buttons on the device.
+To use the controller's inputs, you must [register callback functions](/appendix/apis/components/input-controller/#registercontrolcallback) to the [`Control`s](/appendix/apis/components/input-controller/#control-field) with the [`input` API](/appendix/apis/components/input-controller/).
 
-The callback functions can then handle the [Events](/components/input-controller/#getevents) that are sent when the Control is activated or moved.
+The callback functions can then handle the [Events](/appendix/apis/components/input-controller/#getevents) that are sent when the `Control` is activated or moved.
 For example, when a specific button is pushed, the callback function registered to it can move another component, or print a specific output.
 
-Most machines with an input controller need at least the following hardware:
-
-- A computer capable of running `viam-server`.
-- A power supply cable or batteries for the input device and the machine.
-- A component that you can direct the input to control, like an [arm](/components/arm/) or [motor](/components/motor/).
-
-## Related services
-
-{{< cards >}}
-{{< relatedcard link="/services/base-rc/" >}}
-{{< /cards >}}
+The [base remote control service](/services/base-rc/) implements an input controller as a remote control for a base.
```

Comment:
I think this is a bit vague/not helpful, though of course they can click the link.

Reply:
you're right but I will leave that for when we review the page content more thoroughly.